vars/publishIntegTestResults.groovy

```diff
@@ -54,25 +54,32 @@ void call(Map args = [:]) {
 
     manifest.components.each { component ->
         def componentName = component.name
+        def componentRepo = component.repository.split('/')[-1].replace('.git', '')
+        def componentRepoUrl = component.repository
```

Comment:
I think we should also remove `.git` here as sometimes people didnt add that in input manifest, causing bundle manifest to sometimes have .git sometimes not, which will make your data diverse for the same component at times.

Reply:
I feel like we should also try to remove everything before `github.com` as people tend to mistype https vs http or anything, causing diverse data.

Reply:
so component repo url should just show `opensearch-project/OpenSearch`.a

Reply:
No, just strip `https://` `http://` `.git` and then index.
You dont need to hardcode the opensearch-project.

Reply:
I was thinking index as `https://github.com/opensearch-project/OpenSearch` and using OSD scripted fields just add `opensearch-project/OpenSearch` to reduce the field length for visualizations, but during any query it would return the full URL as  `https://github.com/opensearch-project/OpenSearch`.

Reply:
Its because Rishab, when we browse (just try copy pasting this in browser) `github.com/opensearch-project/OpenSearch`, it should add `https://` and add take us to the repo.